salt/vector/install.sls

```diff
@@ -0,0 +1,15 @@
+install_vector_repo:
+  cmd.run:
+    - name: curl -1sLf 'https://repositories.timber.io/public/vector/cfg/setup/bash.deb.sh' | bash
```

Comment:
Rather than running a curl-bash we should either set up the package repository, or just download the compiled binary, especially since we're creating the systemd unit anyway.

Digging into the script the Apt repository is at
```
deb https://repositories.timber.io/public/vector/deb/debian buster main

deb-src https://repositories.timber.io/public/vector/deb/debian buster main
```

https://vector.dev/docs/setup/installation/manual/from-archives/

Reply:
I wondered about that because I've done so elsewhere, but did not look into the script. It's not doing anything special outside of setting the apt repository like we usually do?

Reply:
FWIW, Vector's installation documentation says to run the script to install it; but I guess they can say whatever they want to make it look easier.

Reply:
Yeah, the script just sets up the apt repo, nothing much beyond that.

Reply:
I've tried to handle this in 60883a26

The script customizes the apt sources list based on the particular version of Debian and the architecture.